### PR TITLE
[Playwright] Fix running with docker

### PR DESCRIPTION
**Pull request by boubkerbribri**

### CODE REVIEW

File: tests/puppeteer/.docker/Dockerfile [hunk 1]
```diff
@@ -1,27 +1,52 @@
...
+# 6. Create folder for tests and copy component
+RUN mkdir -p /tests/reports
 WORKDIR /tests/
 COPY ["./", "./"]
 
-# Install dependencies
-RUN npm install
+# 7. Install dependencies and force install playwright to download browsers
+RUN npm install \
+    && npm i playwright@1.0.2
```
Comment (boubkerbribri) [inline tests/puppeteer/.docker/Dockerfile:52]: Browsers are not downloaded with only `npm install` so i had to run `npm i playwright@1.0.2` to solve the problem.
problem reported here : https://github.com/microsoft/playwright/issues/2386#issuecomment-636703126

Comment (SimonGrn) [reply tests/puppeteer/.docker/Dockerfile:52]: That's weird, never had this problem on my machine

Comment (SimonGrn) [reply tests/puppeteer/.docker/Dockerfile:52]: We will need to update it for every release or what ?

Comment (boubkerbribri) [reply tests/puppeteer/.docker/Dockerfile:52]: I don't know exactly if its a bug or I missed something but i hope for more information on the playwright issue

Comment (khouloudbelguith) [reply tests/puppeteer/.docker/Dockerfile:52]: Hi @SimonGrn @boubkerbribri, I have the same issue on my machine and this solution solved it :+1: 

Thanks!

**Pull request merged**